addons/payment/views/payment_form_templates.xml

```diff
@@ -419,14 +419,19 @@
                 <p t-else="">
                     No payment providers are configured.
                 </p>
+                <t
+                    t-set="onboarding_provider"
+                    t-value="request.env['res.config.settings'].sudo().new({}).onboarding_payment_module"
+                />
```

Comment:
I am not sure it would be good idea to create setting object here. May be instead we could just give static name for stable (19.0 when freeze happen) like `Activate Provider` and then improve it in master by moving it into `res.company`

What do you think @AntoineVDV? or Is it okay to create setting object here in the template 🤔

Reply:
I agree it's not pretty, but how would you know the button should be hidden if no onboarding provider is available?